source/stylesheets/mixins/_lists.scss

```diff
@@ -1,6 +1,5 @@
 @mixin plain-list {
-  &,
-  & li {
+  &, & > li {
```

Comment:
Yeah. By the way, you could remove the second `&` I think. Don't know why i put it there.